portfolio/src/main/webapp/style.css

```diff
@@ -26,7 +26,15 @@ footer {
   color: white;
 }
 
-#padding {
+.padding a{
     padding: 10px;
 }
 
+.img-container img {
+width: 300px;
```

Comment:
ca you make sure the indentation is consistent here and elsewhere in the file?

Reply:
solved